data.md

```diff
@@ -337,23 +347,84 @@ One needs to unname the `ValTypes` first before calling the `#take` or `#drop` f
 Strings
 -------
 
+Wasm use a different character escape rule with K, so we need to define the `unescape` function ourselves.
+
+```k
+    syntax String ::= unescape(String)              [function]
+                    | unescape(String, Int, String) [function, klabel(unescapeAux)]
+ // -------------------------------------------------------------------------------
+    rule unescape(S) => unescape(S, 1, "")
+    rule unescape(S, IDX, SB) => unescape(S, IDX +Int 1, SB +String substrString(S, IDX, IDX +Int 1))
+      requires IDX <Int lengthString(S) -Int 1
+       andBool substrString(S, IDX, IDX +Int 1) =/=K "\\"
+    rule unescape(S, IDX, SB) => unescape(S, IDX +Int 3, SB +String chrChar(String2Base(substrString(S, IDX +Int 1, IDX +Int 3), 16)))
+      requires IDX <Int lengthString(S) -Int 3
+       andBool substrString(S, IDX, IDX +Int 1) ==K "\\"
+       andBool (findChar("0123456789abcdefABCDEF", substrString(S, IDX +Int 1, IDX +Int 2), 0) =/=Int -1 )
+    rule unescape(S, IDX, SB) => SB
+      requires IDX ==Int lengthString(S) -Int 1
+
```

Comment:
... and that the characters "\t", "\n", "\r", "\"", "\'", and "\\" are interpreted as regular escaped ascii symbols, ...